CoqInteractive.sublime-settings

```diff
@@ -17,5 +17,13 @@
     "view_style": "inline",
 
     // When sending commands to Coq, move the cursor to the end of the command.
-    "move_cursor_after_command": true
+    "move_cursor_after_command": true,
+
+    // Goals to show when display the current state:
+    //
+    //  - "primary": the absolute current goal, always only one
+    //
+    //  - "current": all currently-focused goals (like CoqIDE)
+    //
+    "show_goals": "current"
```

Comment:
I don't love the word "current" for this setting. In some sense, the "primary" goal is also the "current" goal. How about "all" instead? (I'm open to suggestions here.)

Reply:
Yeah it feels a little ambiguous. "all" wouldn't be quite right since background, shelved and admitted goals are not shown. Now that I think about it, maybe "focused" makes the most sense?

Reply:
Agreed, I think that's good.

(For what it's worth, I don't think I ever built up the right mental model of focused vs unfocused goals in Coq.)

Reply:
My understanding is that focused goals are the ones you can see in CoqIDE, and which commands like `1:`, `all:` and bullets are based on. This is compatible with the (now deprecated) `Focus` command and it seems to line up with the code.